build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/leafTask.ts

```diff
@@ -567,6 +574,12 @@ export class UnknownLeafTask extends LeafTask {
 	}
 }
 
+/**
+ * A Leaf task base that can be used for tasks that have a list of input and output file paths to include in the
+ * donefile. By default, the donefile will contain the filestat information, like last modified time, as the values in
```

Comment:
optional nit: I prefer this wording personally
```suggestion
 * donefile. By default, the donefile will contain the filestat information (i.e. last modified time) as the values in
```